Gruntfile.js

```diff
@@ -23,22 +22,10 @@ module.exports = function(grunt) {
       production: 'publish',
       bamboo: 'bamboo',
 
-      // webrtc-adapter submodule
-      webrtcAdapterSubmodule: 'third_party/adapter/',
-      googleAdapterPath: '<%= webrtcAdapterSubmodule %>/out/adapter.js',
-
       pluginInfoRoot: grunt.option('pluginInfoRoot') || '<%= source %>',
       pluginInfoFile: grunt.option('pluginInfoFile') || 'pluginInfo.js',
 
-      githash: {
-        submodule: {
-          options: {
-            dir: '<%= webrtcAdapterSubmodule %>'
-          }
-        },
-      },
-
-      version: '<%= pkg.version %>-<%= githash.submodule.short %>',
+      version: '<%= pkg.version %>',
```

Comment:
I'm ok with the version being specified in the package.json as long as it explicitly specified and does not rely on ranges. This PR specifies any minor release greater than 2.0.3 (^2.0.3), that I am not ok with. 

@johache are you comfortable with only sticking to published releases? That does limit us slightly from previously where its commit based, however that may not be an issue and the simplicity may be favored.